dissect/target/plugins/apps/container/container.py

```diff
@@ -22,7 +22,7 @@
     ("datetime", "started"),
     ("datetime", "finished"),
     ("string[]", "ports"),
-    ("string", "names"),
+    ("string", "name"),
```

Comment:
@JSCU-CNI I changed this from `names` to `name`, seemed to be an oversight in naming since it's seemingly always a singular name, but let me know if I'm overlooking anything.